src/iteration_summary.jl

```diff
@@ -34,25 +35,31 @@ inv(sqrt(Γ))`.
 When keyword argument `constrained` is provided with `true` then input `θ`
 is assumed to represent constrained parameters.
 """
-function eki_objective(eki, θ::AbstractVector, G::AbstractVector; constrained = false)
+function eki_objective(eki, θ::AbstractVector, G::AbstractVector; constrained = false, augmented = false)
```

Comment:
Is it important to support `constrained=true`? Might be simpler to support just one usage.

Reply:
I'm fine with removing it though I think it's useful for those who wish to get a feel for the loss landscape given samples that weren't necessarily generated by EKI.